reflex/components/plotly/plotly.py

```diff
@@ -89,7 +93,7 @@ class PlotlyLib(NoSSRComponent):
 
     library = "react-plotly.js@2.6.0"
 
-    lib_dependencies: List[str] = ["plotly.js@2.22.0"]
+    lib_dependencies: List[str] = ["plotly.js@2.22.0", "mergician@2.0.2"]
```

Comment:
I don't think this is needed since you specify the library in `add_imports`. Also maybe we should just combine this class and the bottom class together, since there aren't any other components in PlotlyLib